src/ThemeProvider.tsx

```diff
@@ -52,7 +53,10 @@ export const ThemeProvider: React.FC<ThemeProviderProps> = ({children, ...props}
   const systemColorMode = useSystemColorMode()
   const resolvedColorMode = resolveColorMode(colorMode, systemColorMode)
   const colorScheme = chooseColorScheme(resolvedColorMode, dayScheme, nightScheme)
-  const resolvedTheme = React.useMemo(() => applyColorScheme(theme, colorScheme), [theme, colorScheme])
+  const [resolvedTheme, resolvedColorScheme] = React.useMemo(
+    () => applyColorScheme(theme, colorScheme),
+    [theme, colorScheme]
+  )
```

Comment:
Curious what you're thoughts are about returning an object here instead of an array:

```
const {resolvedTheme, resolvedColorScheme} = ...
// vs
const [resolvedTheme, resolvedColorScheme] = ...
```

Reply:
Either works for me, I was keeping the diff small by returning the tuple, but I don't think there's a reason to do that instead of the object